### ISSUE TITLE
Cannot found the results folder for the pre-trained models

### ISSUE THREAD

**Issue by lijianshe02** [csjunxu/Noisy-As-Clean#4]
You said that the pre-trained models for AWGN denoising on Set12 are provided in the results folder, but the results folder cannot be found in the repo

**Comment by csjunxu** [comment]
Yes, we find that the models are huge, so we did not upload them. The models can be obtained during the training, on the fly. And the test is performed naturally. We have deleted the confusing words.

**Comment by lijianshe02** [comment]
Thanks for you reply. I just want to evaluate the results, so could you please upload the pre-trained model on the cloud disk? Thank you very much.

**Comment by csjunxu** [comment]
Please try our training codes, and the evaluation will be performed after the training. The results are roughly the same as our paper reports. We did not save the pre-trained models (one for each image). Thank you for your understanding.
